src/main/java/com/singh/astha/notificationservice/exceptions/handler/ResponseExceptionHandler.java

```diff
@@ -7,15 +7,18 @@
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 
-
 @ControllerAdvice
 public class ResponseExceptionHandler {
```

Comment:
Let's create an issue to combine all the Exception Handlers, I don't see any advantage of keeping them separate